R/xgboost-development.R

```diff
@@ -23,6 +22,29 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
+#' @param modelName Optional string. Can specify the model name. If used, you must load the same one in the deploy step.
+#' @param xgb_params A list, containing optional xgboost parameters. The full list of params can be found at
```

Comment:
Are these optional xgboost params available to use in deploy? if so, should maybe list them there as well so someone doesn't develop one one, then develop/deploy another.